### ISSUE TITLE
v-model init value

### ISSUE THREAD

**Issue by dimamarksman** [vue-bulma/datepicker#19]
Please implement proper working `v-model` with datepicker to be able to set initial date value.

**Comment by luventa** [comment]
What's your issue? I think it works properly.

`<datepicker v-model="value"></datepicker>`

**Comment by dimamarksman** [comment]
Is there cdn or any public access to the datepicker plugin to show example in a sandbox?

**Comment by rcubitto** [comment]
I have the same problem... 

`<datepicker v-model="value"></datepicker>` if I set `value`, it won't be shown in the input.

**Comment by dimamarksman** [comment]
@luventa what is a reason to assign `v-model="value"` [here](https://github.com/vue-bulma/datepicker/blob/master/src/SingleInput.vue#L2)? 
The `value` is always empty string there that's why the input is always empty in the begining.
I dropped this v-model on my side and use `defaultDate` option ([flatpickr](https://chmln.github.io/flatpickr/)) to assign initial value as a workaround.
But it is not the way the `v-model` directive should work. It should be allowed to put an initial value in the datepicker's `v-model`.

**Comment by luventa** [comment]
Thought you want a init date for it and here is the example code.

```vue
<template>
  <datepicker v-model="value"></datepicker>
</template>

<script>
import Datepicker from 'vue-bulma-datepicker'

export default {
  components: {
    Datepicker
  },

  data () {
    return {
      value: '2016-12-12'
    }
  },
  watch: {
    value (newVal, oldVal) {
      console.log(newVal, oldVal)
    }
  }
}
</script>

```

**Comment by dimamarksman** [comment]
Yes, but using `wrap: true` option.
```
<template>
  <datepicker v-model="value" :config="{ wrap: true }">
    <a class="button" data-toggle><i class="fa fa-calendar"></i></a>
    <a class="button" data-clear><i class="fa fa-close"></i></a>
  </datepicker>
</template>

<script>
import Datepicker from 'vue-bulma-datepicker'

export default {
  components: {
    Datepicker
  },

  data () {
    return {
      value: '2016-12-12'
    }
  },
  watch: {
    value (newVal, oldVal) {
      console.log(newVal, oldVal)
    }
  }
}
</script>
```

**Comment by luventa** [comment]
@dimamarksman I think this is a bug. Will fix

**Comment by dimamarksman** [comment]
Thank you!

21 февр. 2017 г. 12:22 PM пользователь "Pan Yue" <notifications@github.com>
написал:

**Comment by luventa** [comment]
fixed. Will publish soon. Feel free to reopen
Thanks!

**Comment by luventa** [comment]
I have to rebase my commits to keep minimum code change. 
Please check the fix [4dce70f](https://github.com/vue-bulma/datepicker/commit/4dce70f7e44ab24315d803ffa9f38f542c3a57af)

I will publish this component after other issues are fixed. 
Please make change locally for time being. Really appreciate.
